afkak/test/test_consumer.py

```diff
@@ -782,6 +782,20 @@ def test_consumer_error_during_offset(self):
         consumer.stop()
         self.assertEqual(self.successResultOf(d), (None, None))
 
+    def test_consumer_offset_out_of_range_error(self):
+        topic = 'offset_out_of_range_error'
+        part = 911
+        reqs_ds = [Deferred(), Deferred()]
+        mockclient = Mock()
+        mockclient.send_offset_request.side_effect = reqs_ds
+        consumer = Consumer(mockclient, topic, part, Mock())
+        consumer._clock = MemoryReactorClock()
+        consumer.start(OFFSET_LATEST)
+        # Make sure request for offset was made
+        request = OffsetRequest(topic='error_during_offset', partition=part, time=-1, max_offsets=1)
```

Comment:
The topic here looks wrong.  Also, it doesn't seem like the test does anything to test the offset being out of range?